website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "dbt Mesh FAQs"
+description: "Read the FAQs to learn more about dbt Mesh, how it works, compatibility, and more."
+hoverSnippet: "dbt Mesh FAQs"
+sidebar_label: "dbt Mesh FAQs"
+---
+
+dbt Mesh is a new architecture enabled by dbt Cloud. It allows you to better manage complexity by deploying multiple interconnected dbt projects instead of a single large, monolithic project. It’s designed to accelerate development, without compromising governance.
+
+## Overview of Mesh
+
+<detailsToggle alt_header="How do I implement contracts for my models?">
+
+In dbt, [model contracts](/docs/collaborate/govern/model-contracts) serve as a governance tool enabling the definition and enforcement of data structure standards in your dbt models. They allow you to specify and uphold data model guarantees, including column data types, allowing for stability of dependent models. Should a model fail to adhere to its established contracts, it will not successfully build.
+
+</detailsToggle>
+
+<detailsToggle alt_header="What are model versions?">
+
+dbt [model versions](https://docs.getdbt.com/docs/collaborate/govern/model-versions) are iterations of your dbt models made over time. In many cases, you may knowingly choose to change a model’s structure in a way that “breaks” the previous model contract, and may break downstream queries depending on that model’s structure. When you do so, you may find it useful to create a new version of the model to signify this change.
+
+You can use **model versions** to:
+
+- Test "prerelease" changes (in production, in downstream systems)
+- Bump the latest version, to be used as the canonical source of truth
+- Offer a migration window off the "old" version
+
+</detailsToggle>
+
+<detailsToggle alt_header="What are model access modifiers?">
+
+A [model access modifier](/docs/collaborate/govern/model-access) in dbt determines if a model is accessible as an input to other dbt models and projects. It specifies where a model can be referenced using [the `ref` function](https://docs.getdbt.com/reference/dbt-jinja-functions/ref). There are three types of access modifiers:
+
+1. **Private:** A model with a private access modifier is only referenceable by models within the same group. This is intended for models that are implementation details and are meant to be used only within a specific group of related models.
+2. **Protected:** Models with a protected access modifier can be referenced by any other model within the same dbt project or when the project is installed as a package. This is the default setting for all models, ensuring backward compatibility, especially when groups are assigned to an existing set of models.
+3. **Public:** A public model can be referenced across different groups, packages, or projects. This is suitable for stable and mature models that serve as interfaces for other teams or projects.
```

Comment:
Should we mention here that we highly recommend using this sparingly?

Reply:
Hmm good point. I will follow up with @azzam34 to see if this is necessary!

Reply:
Is that something we actually normally recommend as a best practice?

Reply:
I've seen it recommended before. @graciegoheen Should we mention that public models should be used sparingly for specific use cases here?

Reply:
I will publish and we can come back to this! Thanks so much for flagging it, Matt!

Reply:
Will ask in internal slack Monday #tmp-dbt-mesh-external-faq